### PR TITLE
Blog start

### PR DESCRIPTION
**Pull request by E-zh**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -17,7 +17,7 @@
 ```
 * Run `php composer.phar update`
 * Add in your `config/app.php` file in providers:
-```PHP
+``` PHP
 Ourgarage\Blog\BlogServiceProvider::class,
 ```
 * Run `php artisan vendor:publish`
```
Comment (garf) [inline README.md]: Please add the additional parmeters. Ask @Anonim-khb

File: src/resources/assets/css/blog.css [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.admin-dashboard .content-wrapper .blog-index .for-form-inline form {
+  display: inline-block;
+}
```
Comment (garf) [inline src/resources/assets/css/blog.css:3]: Add one blank line below please

File: src/resources/assets/css/blog.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+.admin-dashboard .blog-create .register-box {
+  margin-top: 1em !important;
```
Comment (garf) [inline src/resources/assets/css/blog.css]: Do we need an `!important` here?

File: src/resources/assets/css/blog.less [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.admin-dashboard {
```
Comment (garf) [inline src/resources/assets/css/blog.less:1]: Let's use 4-spaces indentation?

File: src/resources/assets/css/blog.less [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+  .content-wrapper {
+    .blog-index {
+      .for-form-inline {
+        form {
+          display: inline-block;
+        }
+      }
+    }
+  }
+  .blog-create {
```
Comment (garf) [inline src/resources/assets/css/blog.less]: Blank line below :)

File: src/resources/assets/css/blog.less [hunk 6]
```diff
@@ -0,0 +1,16 @@
+.admin-dashboard {
+  .content-wrapper {
+    .blog-index {
```
Comment (garf) [inline src/resources/assets/css/blog.less]: Let's separate every class declaraion with blank line. ;)

File: src/resources/views/admin/settings.blade.php [hunk 7]
```diff
@@ -0,0 +1,18 @@
...
+@endsection
+
+@section('body-title')
+    {{ trans('blog::blog.settings.title') }}
+@endsection
+
+@section('body')
+    <div class="settings-index">
+
+    @include('admin.basis.notifications-page')
```
Comment (garf) [inline src/resources/views/admin/settings.blade.php]: Indentation :)

**Review by garf — CHANGES_REQUESTED** [review]
Small comments.

File: src/resources/assets/css/blog.less (hunk 4 above)
Comment (E-zh) [reply src/resources/assets/css/blog.less:1]: It did Reformat code PhpStorm :)

Comment (garf) [reply src/resources/assets/css/blog.less:1]: So you need to fix the format in Settings -> Editor -> Code Style :)

File: src/resources/views/admin/settings.blade.php [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    {{ trans('blog::blog.settings.title') }}
+@endsection
+
+@section('body')
+    <div class="settings-index">
+
+        @include('admin.basis.notifications-page')
+
+    </div>
+
```
Comment (garf) [inline src/resources/views/admin/settings.blade.php]: Please remove line

File: src/resources/views/basis/css.blade.php [hunk 9]
```diff
@@ -0,0 +1 @@
+<link href='/packages/blog/css/blog.css' rel='stylesheet' type='text/css'>
```
Comment (garf) [inline src/resources/views/basis/css.blade.php]: Nope! :)
This file shouldn't be in global. What if blog is not installed?

**Pull request merged**